### PR TITLE
Route details

### PR DESCRIPTION
**Pull request by japneetbhatia**

### CODE REVIEW

File: src/components/RouteLine.css [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+.show-station-details {
+    font-family: "Roboto","Helvetica","Arial",sans-serif;
+    font-weight: 400;
+    font-size: 0.875rem;
+    line-height: 1.43;
+    color: rgba(0, 0, 0, 0.6);
+}
+
+.css-8t49rw-MuiStepConnector-line {
```
[the remark this answers is not in the document]
Comment (hariharanweb) [reply src/components/RouteLine.css:13]: instead add styled components

File: src/components/RouteLine.css [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    margin-left: 37px !important;
```
Comment (hariharanweb) [inline src/components/RouteLine.css:14]: no obsolute px; relative

File: src/components/RouteLine.js [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+          )}
+        </Grid>
+        <Divider variant="inset" />
+        <Collapse in={open} timeout="auto" unmountOnExit>
+          <Grid className="content">
+            {routeDetail.stops.map((stop) => (
+              <Step key={stop.descriptor.name}>
+                <Grid display="flex">
+                  <Grid paddingTop="10px" paddingRight="7px" className="show-station-details">
+                    {(stop.time.timestamp).substring(11, 16)}
```
Comment (hariharanweb) [inline src/components/RouteLine.js:85]: Wrong. use better approach. Use moment library or get parts of date. Move to separate component to display time

File: src/components/RouteDetails.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import { Grid } from '@mui/material';
+import React from 'react';
+import RouteLine from './RouteLine';
+
+const RouteDetails = ({ routeDetails }) => (
+  <Grid>
+    {routeDetails.map((routeDetail) => (
+      <RouteLine routeDetail={routeDetail} />
```
Comment (hariharanweb) [inline src/components/RouteDetails.js:8]: component name should be RouteDetails instead of RouteLine

File: src/components/RouteLine.css [hunk 5]
```diff
@@ -0,0 +1,24 @@
+.show-station-icon {
+    padding-top: 13%;
+}
+
+.show-station-details {
+    font-family: "Roboto","Helvetica","Arial",sans-serif;
```
Comment (hariharanweb) [inline src/components/RouteLine.css:6]: Is this different from what we see in the app. If yes dont change the font only for this

File: src/components/RouteLine.css [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+    margin-left: 37px !important;
+    min-height: 8px !important;
+}
+
+.css-1db085k-MuiSvgIcon-root {
+    width: 15px !important;
+}
+
+.start-end-station {
+    padding-left: 42px;
```
Comment (hariharanweb) [inline src/components/RouteLine.css:23]: no absolute

Comment (japneetbhatia) [reply src/components/RouteLine.css:23]: updated

File: src/components/RouteLine.js [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+  return (
+    <Grid paddingBottom="20px">
+      <Stepper orientation="vertical">
+        <Step key={routeDetail.startLocation.descriptor.name} className="start-end-station">
+          <StepLabel StepIconComponent={CustomStepIcon}>
+            <Grid display="flex" color="rgba(0, 0, 0, 0.6)">
+              <b>
+                {routeDetail.startLocation.descriptor.name}
+              </b>
+              <Grid paddingLeft="30px" fontSize="0.8rem" className="show-station-details">
```
[the remark this answers is not in the document]
Comment (japneetbhatia) [reply src/components/RouteLine.js:50]: updated

File: src/components/RouteDetails.js (hunk 4 above)
Comment (japneetbhatia) [reply src/components/RouteDetails.js:8]: changed to RouteDetail, RouteDetails is already present
From RouteDetails we are passing every route to RouteDetail(RouteLine)

File: src/components/RouteLine.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (japneetbhatia) [reply src/components/RouteLine.css:13]: Updated

**Pull request merged**